doc/users/dflt_style_changes.rst

```diff
@@ -606,6 +606,7 @@ Legends
        with mpl.rc_context(rc=rcparams):
            x = range(N)
            y = np.cumsum(np.random.randn(N) )
+	   # unpack the single Line2D artist
```

Comment:
Docs are built with Python 2, so any examples using tabs are probably not a big issue, plus this line is a comment anyway.

I don't think PEP8 is run on any inline examples.